dev/devicelab/bin/tasks/flavors_test_ios.dart

```diff
@@ -5,13 +5,38 @@
 import 'package:flutter_devicelab/framework/devices.dart';
 import 'package:flutter_devicelab/framework/framework.dart';
 import 'package:flutter_devicelab/framework/task_result.dart';
+import 'package:flutter_devicelab/framework/utils.dart';
 import 'package:flutter_devicelab/tasks/integration_tests.dart';
 
 Future<void> main() async {
   deviceOperatingSystem = DeviceOperatingSystem.ios;
   await task(() async {
     await createFlavorsTest().call();
     await createIntegrationTestFlavorsTest().call();
+    // test install and uninstall of flavors app
```

Comment:
Does the `flavors_test_ios` change much? If yes, you may want to add `bringup: true` to `Mac_ios flavors_test_ios` for validation.

Reply:
SGTM

Reply:
Thanks for the friendly reminder @keyonghan, I didn't realize that there were other test targets that were affected by this task change.